pkg/daemon/update_test.go

```diff
@@ -20,6 +24,49 @@ import (
 	k8sfake "k8s.io/client-go/kubernetes/fake"
 )
 
+func newMockDaemon() Daemon {
+	// testClient is the NodeUpdaterClient mock instance that will front
+	// calls to update the host.
+	testClient := RpmOstreeClientMock{
+		GetBootedOSImageURLReturns: []GetBootedOSImageURLReturn{},
+	}
+
+	// Create a Daemon instance with mocked clients
+	return Daemon{
+		mock:              true,
+		name:              "nodeName",
+		os:                OperatingSystem{},
+		NodeUpdaterClient: testClient,
+		kubeClient:        k8sfake.NewSimpleClientset(),
+		bootedOSImageURL:  "test",
+	}
+}
+
+func setupTempDirWithEtc(t *testing.T) (string, func()) {
+	// Make a temp dir to put the testing files in, and make sure we clean it up
```

Comment:
Add a `t.Helper()` call at the top of this helper function. This marks this particular function as a test helper which makes reading stack traces much easier if something in here panics.